src/apps/applets/tests/test_applet_activity_items.py

```diff
@@ -2891,6 +2891,23 @@ async def test_create_applet_single_multi_select_response_values_value_null_auto
         option2["value"] = None
         item["response_values"]["options"].append(option2)
         item["response_type"] = response_type
+        resp = await self.client.post(
+            self.applet_create_url.format(
+                owner_id="7484f34a-3acc-4ee6-8a94-fd7299502fa1"
+            ),
+            data=applet_minimal_data,
+        )
+        assert (
+            resp.status_code
+            == activity_errors.DuplicateActivityItemOptionIdError.status_code
+        )
+        assert (
+            resp.json()["result"][0]["message"]
+            == activity_errors.DuplicateActivityItemOptionIdError.message
+        )
+        option2["id"] = None
```

Comment:
Will be better to add uuid. I think we can not have id None by business logic.

Reply:
When id is none, new uuid is given here https://github.com/ChildMindInstitute/mindlogger-backend-refactor/blob/ad7befdc22a2fc947ec7c08f572a37075c2e1581/src/apps/activities/domain/response_values.py#L88